spiketoolkit/validation/quality_metric_classes/utils/thresholdcurator.py

```diff
@@ -1,6 +1,4 @@
 from .curationsortingextractor import CurationSortingExtractor
-import spiketoolkit as st
```

Comment:
maybe the thresholdcurator and curationsortingextractor should be in the curation module?

Reply:
Same issue as above.